### PR TITLE
fix(remove periods from table)

### PR DESCRIPTION
**Pull request by ghost**
Follow up to https://github.com/newrelic/docs-website/pull/13524/files

### CODE REVIEW

File: src/content/docs/licenses/license-information/referenced-policies/security-guide.mdx [hunk 1]
```diff
@@ -432,10 +432,10 @@ to the off-site backup system via Amazon Simple Storage Service (S3).
 * The Services that operate on Amazon Web Services (“AWS”) are protected by the security and environmental controls of AWS. Detailed information about AWS security is available at [https://aws.amazon.com/security/](https://aws.amazon.com/security/) and [http://aws.amazon.com/security/sharing-the-security-responsibility/](http://aws.amazon.com/security/sharing-the-security-responsibility/). Data encryption at rest utilizes FIPS 140-2 compliant encryption methodology. For AWS SOC Reports, please see [https://aws.amazon.com/compliance/soc-faqs/](https://aws.amazon.com/compliance/soc-faqs/).
 * The Services that operate on Google Cloud Platform ("GCP") are protected by the security and environmental controls of GCP. Detailed information about GCP security is available at [https://cloud.google.com/docs/tutorials#security](https://cloud.google.com/docs/tutorials#security). For GCP reports, please see [https://cloud.google.com/security/compliance/](https://cloud.google.com/security/compliance/).
 * The Services in the following data centers are being migrated. The following will be deprecated by July 2024:
```
Comment (ally-sassman) [inline src/content/docs/licenses/license-information/referenced-policies/security-guide.mdx:434]:
```suggestion
* *The Services in the following data centers are being migrated. The following will be deprecated by July 2024: 
```

File: src/content/docs/licenses/license-information/referenced-policies/security-guide.mdx [hunk 2]
```diff
@@ -431,11 +431,11 @@ to the off-site backup system via Amazon Simple Storage Service (S3).
...
 * The Services that operate on Google Cloud Platform ("GCP") are protected by the security and environmental controls of GCP. Detailed information about GCP security is available at [https://cloud.google.com/docs/tutorials#security](https://cloud.google.com/docs/tutorials#security). For GCP reports, please see [https://cloud.google.com/security/compliance/](https://cloud.google.com/security/compliance/).
-* The Services in the following data centers are being migrated. The following will be deprecated by July 2024: 
-* IBM
-* Deft
-* Zayo
-* QTS
+* *The Services in the following data centers are being migrated. The following will be deprecated by July 2024:
```
Comment (ally-sassman) [inline src/content/docs/licenses/license-information/referenced-policies/security-guide.mdx:434]:
```suggestion
* *The Services in the following data centers are being migrated and will be deprecated by July 2024:
```

Comment (ArielRobinson1) [reply src/content/docs/licenses/license-information/referenced-policies/security-guide.mdx:434]: definitely - good call

**Review by ArielRobinson1 — COMMENTED** [review]
Looks good, thanks so much Ally!

**Pull request closed without merging**